addons/csi-vmware-cloud-director/vcloud-csi-config.yaml

```diff
@@ -11,6 +11,5 @@ data:
       host: {{ required "Please provide VCD_URL" (trimSuffix "/api" .Credentials.VCD_URL) }}
       org: {{ required "Please provide VCD_ORG" .Credentials.VCD_ORG }}
       vdc: {{ required "Please provide VCD_VDC" .Credentials.VCD_VDC }}
-      vAppName: {{ .Config.CloudProvider.VMwareCloudDirector.VApp }}
+      vAppName: {{ default .Config.CloudProvider.VMwareCloudDirector.VApp .Params.vApp }}
```

Comment:
Can we call this parameter the full name, i.e. `vAppName`?

Reply:
Sure, done 👍